src/mixins/portfolio.js

```diff
@@ -134,87 +134,73 @@ export default {
     collectedOrderOptions() {
       const options = [
         {
-          value: {
-            orderBy: ORDER_COLLECTED_CLASS_ID_BY.PRICE,
-            order: ORDER.DESC,
-          },
-          name: this.$t('order_menu_price'),
-          format: ORDER_COLLECTED_CLASS_ID_BY.PRICE + ORDER.DESC,
+          value: `${ORDER_COLLECTED_CLASS_ID_BY.PRICE}-${ORDER.DESC}`,
+          name: this.$t('order_menu_by', {
+            order: `${this.$t('order_menu_price')}`,
+          }),
```

Comment:
This part can be refactored into a function that takes `order` as a param and outputs the `name`

Btw, no need to wrap with string literals
```suggestion
          name: this.$t('order_menu_by', {
            order: this.$t('order_menu_price')
          }),
```